src/main/java/com/xiaomi/infra/pegasus/client/ClientOptions.java

```diff
@@ -345,37 +332,24 @@ public Builder metaQueryTimeout(Duration metaQueryTimeout) {
     }
 
     /**
-     * Whether to enable authentication. Defaults to {@literal false}, see {@link
-     * #DEFAULT_ENABLE_AUTH}.
+     * authentiation protocol. Defaults to {@literal ""}, See {@link #DEFAULT_AUTH_PROTOCOL}
```

Comment:
```suggestion
     * The authentiation protocol to use. Available protocols are:
     * <p><ul>
     *  <li>""</li>
     * </ul></p>
     * Defaults to {@literal ""}, See {@link #DEFAULT_AUTH_PROTOCOL}
```

I don't see any available protocols? What does empty string mean?

Reply:
empty means the authentiation is disabled